### ISSUE TITLE
Add a way to skip certain files/models

### ISSUE THREAD

**Issue by westonganger** [7even/active_record-annotate#13]
We need some other way to ignore certain files. Maybe we could have an `.annotate_ignore` file in the project root where we list these?

**Comment by 7even** [comment]
What files do you want to ignore, for example?

**Comment by westonganger** [comment]
Any models I do not wish to be annotated or maybe things like ApplicationRecord and other custom parent classes I may have

**Comment by 7even** [comment]
@westonganger is this still relevant after the ApplicationRecord problem got fixed?

**Comment by westonganger** [comment]
Yes I would say so sometimes we may want to skip certain files on a custom basis for brevity.
